auth/local/auth.controller.js

```diff
@@ -52,7 +58,38 @@ async function verifyAccountHandler(req, res) {
   }
 }
 
+async function changePassword(req, res) {
+  try {
+    const { id } = req.params;
```

Comment:
What do you think if we obtain this data but directly from the user's token, through the `req.user`?

Reply:
```js
const { id } = req.user
```